capstone/src/components/User/Adventurer/Adventurer.jsx

```diff
@@ -1,28 +1,32 @@
 import './Adventurer.css';
 import { useNavigate, useParams } from 'react-router-dom';
-import { useEffect, useState, useMemo } from 'react';
+import { useEffect, useState, useMemo, useContext } from 'react';
 import axios from 'axios';
 import FindAdventure from './FindAdventure/FindAdventure';
 import Preference from './Preference/Preference';
 import Header from '../Header/Header';
 import ExperienceInfo from './ExperienceInfo/ExperienceInfo';
+import UserContext from '../../../Contexts/UserContext';
 
 // CONTEXTS
 import AdventurerContext from '../../../Contexts/AdventurerContext';
 
 const baseURL = process.env.REACT_APP_BASE_URL;
 
-function getNearbyRestaurants(location) {
-  return axios.post(`${baseURL}/adventure/restaurants`, location);
+function getNearbyRestaurants(lat, lng, username) {
+  const values = { lat, lng, username };
+  return axios.post(`${baseURL}/adventure/restaurants`, values);
 }
 
 export default function Adventurer({ setIsLoggedIn, isLoggedIn }) {
+  const { username } = useContext(UserContext);
   // STATES
   const [currentPosition, setCurrentPosition] = useState({ lat: 0, lng: 0 });
   const [isDataFetched, setIsDataFetched] = useState(false);
   const [restaurants, setRestaurants] = useState([]);
 
   // MEMO VALUES
+  // QUESTION: Is memo giving me troubles?
```

Comment:
Can you elaborate on this question?

Reply:
Oh sorry, that was a past question that I actually discovered the answer after the PR. I was going to mention it in our 1:1, but in the end there was no need anymore